www/js/diary/list.js

```diff
@@ -289,14 +289,25 @@ angular.module('emission.main.diary.list',['ui-leaflet',
             if($rootScope.displayingIncident == true && $rootScope.notificationData) {
               console.log('tripgj =>', tripgj);
               console.log('$rootScope.notificationData =>', $rootScope.notificationData);
+              console.log("Delta start_ts = "+
+                ($rootScope.notificationData.start_ts - tripgj.data.properties.start_ts)
+                +" and delta end_ts = "+
```

Comment:
One of the output shows a `negative` delta which I suspect it is the trip we are looking for but I am not certain how should I refer to it. I am started to think that `start_ts` and `end_ts` from `notification` is somewhat not related to the trip but might be related to the time the notification triggered or the time I tap on the notification.